### DISCUSSION TITLE
PUT et PATCH Anomalie Adresse semblent redondants

### DISCUSSION THREAD

**Discussion by alexisthethe** [before-interop/anomalieAdresse#11] [Q&A]
Pour mettre à jour une anomalie adresse, il y a 2 endpoints possibles :

- PUT /{anomalieAdresseId}
- PATCH /{anomalieAdresseId}

Cela me semble redondant et porte à confusion sur l'utilisation de l'API. Ne faudrait-il pas plutôt choisir la méthode qui sera utilisée plutôt que de laisser plusieurs possibilités ?
D'autant plus que j'ai l'impression que dans la réalité on aura besoin uniquement du PATCH (lorsqu'un OC va mettre un jour un ticket il va mettre à jour partiellement le ticket à chaque fois, il ne va jamais faire un PUT).

Qu'en pensez-vous ? @ggrebert

**Comment by ggrebert** [comment]
Je n'ai pas vraiment d'avis sur le sujet.
On peut proposer la suppression du **PUT** à la prochaine réu de septembre si tu veux.

**Reply by alexisthethe** [reply]
Oui ça serait bien je pense. Merci pour ta réponse.

Je peux lancer un sondage pour avoir un ressenti de l'avis de tout le monde avant la réunion de septembre si ça te va

**Reply by alexisthethe** [reply]
Voir cette discussion pour le sondage : https://github.com/before-interop/anomalieAdresse/discussions/14
